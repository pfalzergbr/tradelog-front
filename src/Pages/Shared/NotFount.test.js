import { shallow } from 'enzyme';
import NotFound from './NotFound';
import React from 'react';

it('should render NotFound Component', () => {
    expect(shallow(<NotFound />)).toMatchSnapshot();
})