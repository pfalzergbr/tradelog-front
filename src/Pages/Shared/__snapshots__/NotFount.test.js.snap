// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFound Component 1`] = `
<div>
  404
</div>
`;
